Sources/QR8bitByte.swift

```diff
@@ -29,9 +29,9 @@ struct QR8bitByte {
     let data: String
     let parsedData: Data
     
-    init?(_ data: String) {
+    init?(_ data: String, encodingMode: String.Encoding = .utf8) {
```

Comment:
Can we rename `encodingMode` to `encoding`?

Reply:
Sure, its shorter. I did not put that much thought on the name of this one.